### PR TITLE
[READY]Replaces tools in the syndicate toolbox with syndicate exclusives.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request

Replaces the wrench, crowbar, and wirecutters with syndicate versions with unique sprites and faster tool speed. The screwdriver was always special and used for breaking open the nuke and already has increased tool speed.

image

I felt making a syndicate multitool was unnecessary but I would be happy to do so.

## Why It's Good For The Game
 Makes the item more distinct from the ancient toolbox making it a more loud and flashy alternative to the toolbox filled with legal items that serve the same purpose.

## Changelog
:cl: itseasytosee
tweak: The syndicate toolbox now has unique syndicate tools rather than normal ones.
/:cl:

### CODE REVIEW

File: code/game/objects/items/theft_tools.dm [hunk 1]
```diff
@@ -263,3 +263,24 @@
 	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
 	QDEL_NULL(sliver)
 	update_icon()
+
+//Don't know where else to put fancy syndicate tools
```
Comment (GunHog) [inline code/game/objects/items/theft_tools.dm]: "tgstation/code/game/objects/items/tools/" is a folder you could use to store the new tools in their respective files. This file is for the tools related to stealing the nuke core and Supermatter piece.

**Comment by Farquaar** [comment]
A dark, sinister multitool with an ominous glowing screen would look pretty cool, I think. You should go for it if you've got the time.

**Comment by HomocidlePancakes** [comment]
Synditoolbox tools already had an increased speed.

**Comment by cacogen** [comment]
the sprites aren't good

**Comment by Arathianen** [comment]
I like the sprites, although the wrench needs some touch up.

Please do re-colour the multitool to fit with the others' theme though! It looks very out of place.

File: code/game/objects/items/theft_tools.dm [hunk 2]
```diff
@@ -263,3 +263,24 @@
...
+
+/obj/item/wrench/syndie
+	name = "suspicious-looking wrench"
+	desc = "It’s one of those fancy wrenches that you turn backward without twisting the bolt for faster action."
+	icon_state = "wrench_syndie"
+	toolspeed = 0.5
+
+/obj/item/crowbar/syndie
```
Comment (Fikou) [inline code/game/objects/items/theft_tools.dm]: this crowbar has less force than the red crowbar which was in the toolbox previously

**Comment by optimumtact** [comment]
dont' buff the speed further

File: code/game/objects/items/theft_tools.dm (hunk 2 above)
Comment (optimumtact) [reply code/game/objects/items/theft_tools.dm]: that's very unlikely to come up

Comment (ATH1909) [reply code/game/objects/items/theft_tools.dm]: This is a black and RED crowbar (and a cool/advanced syndicate one at that), so we should give it at least the same force as a normal red crowbar.

**Comment by itseasytosee** [comment]
Multitool time!

File: code/game/objects/items/theft_tools.dm (hunk 1 above)
Comment (itseasytosee) [reply code/game/objects/items/theft_tools.dm]: Thanks, I did not know if that was a good spot either.

**Comment by itseasytosee** [comment]
@HomocidlePancakes the wrench and wirecutters do not they are normal.
And I believe the crowbar is the same found in emergency toolboxes.

**Comment by itseasytosee** [comment]
There is now a multitool sprite and I have fixed the wrench sprite to be more easy on the eyes.

File: code/game/objects/items/theft_tools.dm (hunk 2 above)
Comment (optimumtact) [reply code/game/objects/items/theft_tools.dm]: You're bikeshedding

**Comment by CRITAWAKETS** [comment]
[device.dmi](http://www.mediafire.com/view/iifvmc097z1p0y2/device.dmi)
[tools.dmi](http://www.mediafire.com/view/pfk38ifbok11ety/tools%20(2).dmi)
Made some tweaks to the syndie tools which i think look good.

**Comment by itseasytosee** [comment]
Nice!

**Comment by itseasytosee** [comment]
red tipped combat gloves Y/N?

**Comment by ExcessiveUseOfCobblestone** [comment]
so you can't buy a set of tools without having them scream "I'M A BADDIE"? I can dump the toolbox but If I bought the tools I'd like to keep them. Seems like now if I need to buy this I need to hide everything after which is kinda sad.

Sprites are always appreciated though.

**Comment by itseasytosee** [comment]
@ExcessiveUseOfCobblestone have you looked at the ancient toolbox?

**Comment by trollbreeder** [comment]
@itseasytosee You do know that the ancient toolbox is an assistant-only traitor item, right?
image

**Comment by itseasytosee** [comment]
Well nevermind then

**Comment by CRITAWAKETS** [comment]
The screwdriver of the regular toolbox is already a baddie screwdriver though, and its one of the most used tools in construction.

also combat gloves

**Comment by itseasytosee** [comment]
Would red-tipped combat glove be a neat idea? they look just like black gloves.

**Comment by necromanceranne** [comment]
Combat gloves are kind of meant to look like black gloves except for name. While their mob sprite should stay the same, you could change the icon so they're identifiable when pulled off their wearer.

That said, I don't understand Cobby's concerns much. The tools can be kept in a belt, in a box, whatever, and if you get discovered with the black screwdriver you likely are already caught red-handed. The screwdriver is a giveaway to anyone who knows what it is, not just the toolbox.

Also, storage implants can help hide sensitive items.

As for the sprites I'm having a bit of trouble seeing them against the hud. Maybe make the head of the wrench, the head of the crowbar and the inner blade edge of the wirecutters more silvered so I can actually see them.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm thinking getting caught in the middle of using the tool and uh oh visual difference on examine! If you're in a position where you got searched it can't be helped.

I personally wouldn't mind both existing, with the high profile one giving the toolspeed and the other one just giving the tools (which with recent PRs have become harder to access for some people). There are times you may want to drop the obvious "uh oh im a baddie" gear but still obviously want to have a set of tools to work with.

**Comment by itseasytosee** [comment]
again the screwdriver and gloves are already obvious but an easy solution would be to make the ancient toolbox unrestricted.

**Comment by ExcessiveUseOfCobblestone** [comment]
I can pay for a new screwdriver and don't need the gloves nearly as much if I know the wires, lot easier than having to pay for a full toolset which is what I thought the purpose of the box was for (that and having near-instant access to a full set of tools at a price).

I could care less about the approach, the end goal should be a box that trades the high toolspeed for "kill me" objs and another that makes it much easier to maintain plausible deniability that you're a baddie. If I had to pick between one, I'd pick the latter.

**Comment by itseasytosee** [comment]
So are you implying we should unrestrict the ancient toolbox?

**Comment by ExcessiveUseOfCobblestone** [comment]
If the ancient tool box is a normal toolbox, sure. I don't know what it does.

**Comment by itseasytosee** [comment]
It's a normal toolbox with insuls instead of an analyzer and has a gimmick where it does more damage if you put TC in it

**Comment by itseasytosee** [comment]
It's a normal toolbox with insuls instead of an analyzer and has a gimmick where it does more damage if you put TC in it

**Comment by cacogen** [comment]
based

**Pull request closed without merging**